### DISCUSSION TITLE
[Question / Feature] Support resumable upload

### DISCUSSION THREAD

**Discussion by prabhah** [Studio-42/elFinder#3309] [Ideas]
Does elFinder support resumable file upload?
If not, does anybody knows how to realize this feature?
Thanks ~

**Comment by nao-pon** [comment]
@prabhah Currently, elFinder does not support resumeable upload. We welcome the PR of that function.

**Comment by CodeLyokoXtEAM** [comment]
@nao-pon 
https://github.com/Studio-42/elFinder/issues/1357#issuecomment-394141094
